### PR TITLE
Update cta_names_mappings.md

### PR DESCRIPTION
**Pull request by KadeMorton**

### CODE REVIEW

File: cta_names_mappings.md [hunk 1]
```diff
@@ -1,169 +1,621 @@
...
+* Name = Molerats
+* Full aliases = [Gaza Cybergang Group1](https://cybersecurity.att.com/blogs/labs-research/alien-labs-2019-analysis-of-threat-groups-molerats-and-apt-c-37)
+* Parent group = [Gaza Cybergang](https://cybersecurity.att.com/blogs/labs-research/alien-labs-2019-analysis-of-threat-groups-molerats-and-apt-c-37)
+* Campaigns = [SneakyPastes](https://cybersecurity.att.com/blogs/labs-research/alien-labs-2019-analysis-of-threat-groups-molerats-and-apt-c-37)
+
+---
+
+* Name = Desert Falcon
+* Full aliases = [Arid Viper, APT-C-23](https://malpedia.caad.fkie.fraunhofer.de/actor/aridviper), [Two-Tailed Scorpion](https://www.technadu.com/apt-c-23-re-emerged-front-scene-mygram-hacks/102145/), [Gaza Cybergang Group2](https://cybersecurity.att.com/blogs/labs-research/alien-labs-2019-analysis-of-threat-groups-molerats-and-apt-c-37)
+* Parent group = [Gaza Cybergang](https://cybersecurity.att.com/blogs/labs-research/alien-labs-2019-analysis-of-threat-groups-molerats-and-apt-c-37)
```
Comment (jecarr) [inline cta_names_mappings.md:483]: There's repetition of information by introducing both `Parent group` and `Subgroup` fields.

- You have `Desert Falcon` here with Parent group `Gaza Cybergang`
- In `Gaza Cybergang`, you have the subgroup `Desert Falcons/Gaza Cybergang Group2`
   - Falcon or Falcons?
   - Comma separated values are what I'd be using to update the accompanying JSON file. You've referred to Desert Falcon here by its name plus an alias: name/alias
   - The hyperlink is the same
- You are therefore repeating the parent-sub group relationship in two places as well as the alias name in two places. This means more places you have to manage.
- This also has been done inconsistently:
   - You have  `Parent group = APT41` in two places but no entry for APT41 itself

If it helps you see relationships quickly when looking up an entry then we can leave it, just note that for the JSON file I will not be repeating these fields (I'll probably only list subgroups and not double-back and refer to parent-groups too)

File: cta_names_mappings.md [hunk 2]
```diff
@@ -1,169 +1,621 @@
...
+* Name = APT34
+
+---
+
+* Name = CHRYSENE
+
+---
+
+* Name = APT35
+* Subgroups = [DEV-0270/Nemesis Kitten](https://www.microsoft.com/security/blog/2022/09/07/profiling-dev-0270-phosphorus-ransomware-operations/)
```
Comment (jecarr) [inline cta_names_mappings.md:226]: As stated in another comment, I'm working off of comma separated values. I will therefore consider `DEV-0270/Nemesis Kitten` as one group name

File: cta_names_mappings.md [hunk 3]
```diff
@@ -1,169 +1,621 @@
...
+
+---
+
+* Name = APT-C-21
+* Full aliases = [GnatSpy, FrozenCell, VAMP](https://securityaffairs.co/wordpress/125010/apt/apt-c-23-middle-east-android-spyware.html)
+
+---
+
+* Name = APT-C-35
+* Aliases = [Donot Team, SectorE02](https://www.welivesecurity.com/2022/01/18/donot-go-do-not-respawn/), [SideWinder](https://blog.talosintelligence.com/2022/02/whats-with-shared-vba-code.html), [Rattlesnake, Hardcore Nationalist, RAZOR TIGER, T-APT-04, APT-C-17](https://blog.group-ib.com/sidewinder-antibot)
```
[the remark this answers is not in the document]
Comment (KadeMorton) [reply cta_names_mappings.md:368]: I've now unified these. I've replaced Aliases with Full aliases and Partial aliases due to the nature of how some groups are named.

File: cta_names_mappings.md (hunk 1 above)
Comment (KadeMorton) [reply cta_names_mappings.md:483]: There is repetition for parent and subgroups. This is deliberate. Lets say Candy Corn and Pink Rainbow are two groups. Candy Corn is the parent group and Pink Rainbow is the subgroup. If I look at the entry for Pink Rainbow and nothing else, I should be able to see that they are subgroup of Candy Corn. I shouldn't have to trawl through the list to come across Candy Corn and see Pink Rainbow listed as a subgroup. If nothing is listed under Pink Rainbow and that's all I look at I don't know that relationship exists.

Right now we don't need the software to know about those connections. Down the track we can leverage it but right now this is purely for human readability to understand the relationships, and to get more keywords in for searching.

While we have a whole bunch of different fields, right now all the software needs to use them for are searching keywords.

I've corrected the plural on Falcon.

I've removed the /s.

Don't stress about the hyperlinks being the same.

That is fine as far as the JSON not repeating information.

File: cta_names_mappings.md (hunk 2 above)
Comment (KadeMorton) [reply cta_names_mappings.md:226]: I've removed the /.

File: cta_names_mappings.md [hunk 4]
```diff
@@ -1,169 +1,621 @@
...
+
+---
+
+* Name = CERIUM
+* Country = [North Korea](https://query.prod.cms.rt.microsoft.com/cms/api/am/binary/RWMFIi)
+
+---
+
+* Name = APT38
+* Full aliases = check they are actually full, [Lazarus Group](https://www.fireeye.com/current-threats/apt-groups.html#north-korea), [Hidden Cobra, Guardians of Peace](https://www.cisa.gov/uscert/ncas/alerts/TA17-164A), [NICKEL ACADEMY](https://www.secureworks.com/about/press/media-alert-secureworks-discovers-north-korean-cyber-threat-group-lazarus-spearphishing), [Black Artemis, COVELLITE, CTG-2460, Dark Seoul, High Anonymous, Labyrinth Chollima, New Romanic Cyber Army Team, NNPT Group, Who Am I?, Whois Team, ZINC](https://www.secureworks.com/research/threat-profiles), [TA404](https://www.proofpoint.com/us/blog/threat-insight/above-fold-and-your-inbox-tracing-state-aligned-activity-targeting-journalists), [APT-C-26](https://mp-weixin-qq-com.translate.goog/s/W4hkBRJnwN1G32QCpaNNoA?_x_tr_sl=auto&_x_tr_tl=en&_x_tr_hl=en&_x_tr_pto=wapp&utm_source=substack&utm_medium=email)
```
[the remark this answers is not in the document]
Comment (KadeMorton) [reply cta_names_mappings.md:268]: That's a copy and paste error, I did my check and I've removed the comment.

File: cta_names_mappings.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (jecarr) [reply cta_names_mappings.md]: Thanks!

File: cta_names_mappings.md (hunk 2 above)
Comment (jecarr) [reply cta_names_mappings.md]: Thanks!

File: cta_names_mappings.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (jecarr) [reply cta_names_mappings.md]: Sure, thank you

File: cta_names_mappings.md (hunk 1 above)
Comment (jecarr) [reply cta_names_mappings.md:484]: Oh ok, thanks for the clarification. I had the perspective of data entry and management via the JSON file whereas you've explained the rationale for needing to lookup single entries at a time. I would argue that you can Ctrl+F in seconds to find relationships (and not consider this trawling through) as that's what I had to do when converting your changes to update the JSON file. But as the markdown file will be your remit, if you're happy that works for you then let's stick with it.

**Review by jecarr — APPROVED** [review]
Thanks for the quick replies and quick actioning on my review comments.

**Pull request merged**